docs/dictionary/command/export-snapshot.lcdoc

```diff
@@ -66,10 +66,8 @@ maskFile:
 Specifies the name and location of a file to export as an image mask.
 You can use a maskFile only when exporting in PBM format (paint).
 
-effect:
-Specifies whether graphic effects and a blendLevel should be applied to
-the object before rendering export. ...: Denotes the usual possible
-suffixes for the export snapshot command.
+suffix:
+The usual possible suffixes for the export snapshot command.
```

Comment:
It's not very clear is it? What you've done is an improvement from the Travis point of view, but I've filed a bug report so it doesn't get forgotten:
http://quality.livecode.com/show_bug.cgi?id=18656